data/sql/updates/db_world/2020_04_28_01.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE `command` SET `name`='spect leave', `security`='0', `help`='Syntax: .spect leave\r\nLeave spectator mode.' WHERE (`name`='spect leave');
+UPDATE `command` SET `name`='spect reset', `security`='0', `help`='Syntax: .spect reset\r\nSend addon data.' WHERE (`name`='spect reset');
+UPDATE `command` SET `name`='spect spectate', `security`='0', `help`='Syntax: .spect spectate #player\r\nAllow player to spectate arena from another player.' WHERE (`name`='spect spectate');
+UPDATE `command` SET `name`='spect version', `security`='0', `help`='Syntax: .spect version\r\nCheck version of arena spectator addon.' WHERE (`name`='spect version');
+UPDATE `command` SET `name`='spect watch', `security`='0', `help`='Not used.' WHERE (`name`='spect watch');
```

Comment:
Please don't use `''` around numbers

```suggestion
UPDATE `command` SET `name`='spect leave', `security`=0, `help`='Syntax: .spect leave\r\nLeave spectator mode.' WHERE (`name`='spect leave');
UPDATE `command` SET `name`='spect reset', `security`=0, `help`='Syntax: .spect reset\r\nSend addon data.' WHERE (`name`='spect reset');
UPDATE `command` SET `name`='spect spectate', `security`=0, `help`='Syntax: .spect spectate #player\r\nAllow player to spectate arena from another player.' WHERE (`name`='spect spectate');
UPDATE `command` SET `name`='spect version', `security`=0, `help`='Syntax: .spect version\r\nCheck version of arena spectator addon.' WHERE (`name`='spect version');
UPDATE `command` SET `name`='spect watch', `security`=0, `help`='Not used.' WHERE (`name`='spect watch');
```

Reply:
Edited

Seems .spectate doesn't work. Says "There's no such command"
Thats why i used just .spect